lib/protobuf/nats/client.rb

```diff
@@ -65,6 +71,12 @@ def send_request
         # Nats response timeout.
         retry if (retries -= 1) > 0
         raise
+      rescue NackError
```

Comment:
Would it be worth avoiding all together and just returning one of the symbols `:ack_timeout`, `:nack`, `:resp_timeout` when those conditions occur?